### PR TITLE
Events doc comments

### PR DESCRIPTION
**Pull request by nightskylark**

### CODE REVIEW

**Review by ovchinnikov — DISMISSED** [review]

File: js/events.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+* @publicName data
+* @type object
+*/
+
+/**
+* @name eventToRename
+* @publicName eventToRename
+* @type dxEvent|jQery.Event
+* @return dxEvent
+*/
```
Comment (IlyaKhD) [inline js/events.js]: @return is not applicable at this point. Not a function.

File: js/events.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+/**
+* @name dxEventFields_data
+* @publicName data
+* @type object
+*/
+
+/**
+* @name eventToRename
+* @publicName eventToRename
+* @type dxEvent|jQery.Event
```
Comment (IlyaKhD) [inline js/events.js]: Mistype `jQuery.Event`

File: js/events/drag.js [hunk 3]
```diff
@@ -310,15 +310,15 @@ var DragEmitter = GestureEmitter.inherit({
 /**
  * @name ui events_dxdragstart
  * @publicName dxdragstart
- * @type EVENT
+ * @type classEventType
```
Comment (lukyanovas) [inline js/events/drag.js]: eventType

File: js/events/drag.js [hunk 4]
```diff
@@ -310,15 +310,15 @@ var DragEmitter = GestureEmitter.inherit({
...
  * @type_function_param1 event:jQuery.event
  * @type_function_param1_field1 cancel:boolean
  * @module events/drag
 */
 /**
   * @name ui events_dxdrag
   * @publicName dxdrag
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/drag.js:321]: eventType

File: js/events/drag.js [hunk 5]
```diff
@@ -327,7 +327,7 @@ var DragEmitter = GestureEmitter.inherit({
 /**
   * @name ui events_dxdragend
   * @publicName dxdragend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/drag.js:330]: eventType

File: js/events/transform.js [hunk 6]
```diff
@@ -172,15 +172,15 @@ var TransformEmitter = Emitter.inherit({
 /**
  * @name ui events_dxtransformstart
  * @publicName dxtransformstart
- * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:175]: eventType

File: js/events/transform.js [hunk 7]
```diff
@@ -172,15 +172,15 @@ var TransformEmitter = Emitter.inherit({
...
+ * @type classEventType
  * @type_function_param1 event:jQuery.event
  * @type_function_param1_field1 cancel:boolean
  * @module events/transform
 */
 /**
   * @name ui events_dxtransform
   * @publicName dxtransform
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:183]: eventType

File: js/events/transform.js [hunk 8]
```diff
@@ -194,7 +194,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxtransformend
   * @publicName dxtransformend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:197]: eventType

File: js/events/transform.js [hunk 9]
```diff
@@ -209,15 +209,15 @@ var TransformEmitter = Emitter.inherit({
 /**
  * @name ui events_dxtranslatestart
  * @publicName dxtranslatestart
- * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:212]: eventType

File: js/events/transform.js [hunk 10]
```diff
@@ -209,15 +209,15 @@ var TransformEmitter = Emitter.inherit({
...
+ * @type classEventType
  * @type_function_param1 event:jQuery.event
  * @type_function_param1_field1 cancel:boolean
  * @module events/transform
 */
 /**
   * @name ui events_dxtranslate
   * @publicName dxtranslate
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:220]: eventType

File: js/events/transform.js [hunk 11]
```diff
@@ -227,7 +227,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxtranslateend
   * @publicName dxtranslateend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:230]: eventType

File: js/events/transform.js [hunk 12]
```diff
@@ -238,7 +238,7 @@ var TransformEmitter = Emitter.inherit({
 /**
  * @name ui events_dxzoomstart
  * @publicName dxzoomstart
- * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:241]: eventType

File: js/events/transform.js [hunk 13]
```diff
@@ -247,7 +247,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxzoom
   * @publicName dxzoom
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:250]: eventType

File: js/events/transform.js [hunk 14]
```diff
@@ -258,7 +258,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxzoomend
   * @publicName dxzoomend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:261]: eventType

File: js/events/transform.js [hunk 15]
```diff
@@ -270,15 +270,15 @@ var TransformEmitter = Emitter.inherit({
 /**
 * @name ui events_dxpinchstart
 * @publicName dxpinchstart
-* @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:273]: eventType

File: js/events/transform.js [hunk 16]
```diff
@@ -270,15 +270,15 @@ var TransformEmitter = Emitter.inherit({
...
+* @type classEventType
 * @type_function_param1 event:jQuery.event
 * @type_function_param1_field1 cancel:boolean
 * @module events/transform
    */
 /**
   * @name ui events_dxpinch
   * @publicName dxpinch
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:281]: eventType

File: js/events/transform.js [hunk 17]
```diff
@@ -288,7 +288,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxpinchend
   * @publicName dxpinchend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:291]: eventType

File: js/events/transform.js [hunk 18]
```diff
@@ -299,15 +299,15 @@ var TransformEmitter = Emitter.inherit({
 /**
  * @name ui events_dxrotatestart
  * @publicName dxrotatestart
- * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:302]: eventType

File: js/events/transform.js [hunk 19]
```diff
@@ -299,15 +299,15 @@ var TransformEmitter = Emitter.inherit({
...
+ * @type classEventType
  * @type_function_param1 event:jQuery.event
  * @type_function_param1_field1 cancel:boolean
  * @module events/transform
 */
 /**
   * @name ui events_dxrotate
   * @publicName dxrotate
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:310]: eventType

File: js/events/transform.js [hunk 20]
```diff
@@ -317,7 +317,7 @@ var TransformEmitter = Emitter.inherit({
 /**
   * @name ui events_dxrotateend
   * @publicName dxrotateend
-  * @type EVENT
```
Comment (lukyanovas) [inline js/events/transform.js:320]: eventType

File: js/ui/tree_view/ui.tree_view.base.js [hunk 21]
```diff
@@ -287,7 +287,7 @@ var TreeViewBase = HierarchicalCollectionWidget.inherit({
             element: jQuery.fn.init[1]
             itemData: Object
             itemElement: jQuery.fn.init[1]
-            jQueryEvent: jQuery.Event
+            jQueryEvent: eventType
```
Comment (lukyanovas) [inline js/ui/tree_view/ui.tree_view.base.js]: I think all this comment block should be removed

File: js/events/drag.js (hunk 3 above)
Comment (nightskylark) [reply js/events/drag.js]: Done.

File: js/events/drag.js (hunk 4 above)
Comment (nightskylark) [reply js/events/drag.js:321]: Done.

File: js/events/drag.js (hunk 5 above)
Comment (nightskylark) [reply js/events/drag.js:330]: Done.

File: js/events/transform.js (hunk 6 above)
Comment (nightskylark) [reply js/events/transform.js:175]: Done.

File: js/events/transform.js (hunk 7 above)
Comment (nightskylark) [reply js/events/transform.js:183]: Done.

File: js/events/transform.js (hunk 8 above)
Comment (nightskylark) [reply js/events/transform.js:197]: Done.

File: js/events/transform.js (hunk 9 above)
Comment (nightskylark) [reply js/events/transform.js:212]: Done.

File: js/events/transform.js (hunk 10 above)
Comment (nightskylark) [reply js/events/transform.js:220]: Done.

File: js/events/transform.js (hunk 11 above)
Comment (nightskylark) [reply js/events/transform.js:230]: Done.

File: js/events/transform.js (hunk 12 above)
Comment (nightskylark) [reply js/events/transform.js:241]: Done.

File: js/events/transform.js (hunk 13 above)
Comment (nightskylark) [reply js/events/transform.js:250]: Done.

File: js/events/transform.js (hunk 14 above)
Comment (nightskylark) [reply js/events/transform.js:261]: Done.

File: js/events/transform.js (hunk 15 above)
Comment (nightskylark) [reply js/events/transform.js:273]: Done.

File: js/events/transform.js (hunk 16 above)
Comment (nightskylark) [reply js/events/transform.js:281]: Done.

File: js/events/transform.js (hunk 17 above)
Comment (nightskylark) [reply js/events/transform.js:291]: Done.

File: js/events/transform.js (hunk 18 above)
Comment (nightskylark) [reply js/events/transform.js:302]: Done.

File: js/events/transform.js (hunk 19 above)
Comment (nightskylark) [reply js/events/transform.js:310]: Done.

File: js/events/transform.js (hunk 20 above)
Comment (nightskylark) [reply js/events/transform.js:320]: Done.

File: js/ui/tree_view/ui.tree_view.base.js (hunk 21 above)
Comment (nightskylark) [reply js/ui/tree_view/ui.tree_view.base.js]: Done.

**Review by viterobk — DISMISSED** [review]

**Review by lukyanovas — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**